docs/openapi.json

```diff
@@ -2325,6 +2325,12 @@
                             "type": "string"
                         }
                     },
+                    "subtotals": {
```

Comment:
Wouldn't be better to include explicit values directly? Same format as `total_items`. We could set them -1 when not used. Probably it would be easier to use for frontend guys. @jiridostal @mkholjuraev what do you mean?

Reply:
I did it this way because meta part of API json is the same for all APIs and currently only /systems contains these subtotals.
So others may omit this part completely.
Moreover if we introduce a different set of subtotals, e.g. for /advisories, they will smoothly fit into this dictionary.